all-requirements.txt

```diff
@@ -10,3 +10,6 @@ tornado==4.2
 
 # For simpl.middleware.cors
 WebOb==1.4.1
+
+# MongoDB
+pymongo==2.8.1
```

Comment:
I think, in this particular case, we are porting outdated knowledge and therefore should not do it. Absolutely bring the learnings forward, just not the code. I'm wondering how many things we solved in the past have since been solved in pymongo?

I completely agree with @samstav: if it doesn't work with the latest pymongo, we should not include it. Further I'm guessing that, if the code were updated to be 3.0.3 compatible, it would be very difficult to identify which parts of this code are still necessary.